src/script/interpreter.cpp

```diff
@@ -1584,6 +1604,19 @@ bool ScriptMachine::Step()
                         valtype &vchSig = stacktop(-isig);
                         valtype &vchPubKey = stacktop(-ikey);
 
+                        // If schnorr is enabled, then no signature can be 64 + 1 bytes because multisig does
+                        // not support schnorr, and all 64 byte signatures are assumed to be schnorr.
+                        if (flags & SCRIPT_ENABLE_SCHNORR)
+                        {
+                            if (vchSig.size() == 65) // 64 sig length plus 1 sighashtype
```

Comment:
wonder if we should make the sig length a constant, all the 64s and 65s could lead to human error